src/instructlab/model/evaluate.py

```diff
@@ -333,6 +333,12 @@ def launch_server(
     default="",
     help="TLS client certificate password for model serving.",
 )
+@click.option(
+    "--background",
```

Comment:
This is really:

--background-serving
or
-- verbose-serving
or
--enable-serving-output

Reply:
I like --verbose-serving personally. I want to know what's going on with vLLM or llama.cpp since they might take a while to boot up, but I really only want it as a debug option.

Reply:
I prefer `--enable-serving-output` because it accurately reflects that the flag enables the output of the serving engine. To me, `--verbose-serving` implies that the serving output is already present and will become more verbose if the flag is turned on.

Reply:
I'm going to go with `--enable-serving-output` for the flag for evaluate.